### DISCUSSION TITLE
Thoughts and suggestions on a  few minor things I uncovered in the editor

### DISCUSSION THREAD

**Discussion by iamsaumya** [outline/rich-markdown-editor#573] [Q&A] [answered]
I was tinkering with the editor and uncovered the following:

1. Inside blockquote, we can't directly convert a bullet/numbered list to a checklist. We have to unselect bullet/numbered list then select checklist.

   PS: I know, nobody in their right mind would use a checklist inside blockquote but still...

2. Inside blockquote, if we select the text to be a heading. The heading annotation and blockquote lines overlap. Maybe we can do something about it? 

3. If we type `/@xyz xyz` the BlockMenu stays active.  We have two options here 1. Don't show the menu at all 2. Update the regex to handle this text and show "No results"

4. Suggestion - When the BlockMenu is active, we should handle `Shit-Tab` to move select upwards. 

5.  Inside the table, The highlight menu has two seperators. 

As usual, let me know your thoughts on which are genuine issues to consider. 

Thanks!

**Comment by tommoor** [comment, marked as answer]
> Inside blockquote, we can't directly convert a bullet/numbered list to a checklist. We have to unselect bullet/numbered list then select checklist.

I think this is the case in more spots than just a blockquote – checklists are somewhat unique so it can be tricky to convert in a number of scenarios.

> Inside blockquote, if we select the text to be a heading. The heading annotation and blockquote lines overlap. Maybe we can do something about it?

Sure, that would be nice – it's mostly just css.

> If we type /@xyz xyz the BlockMenu stays active. We have two options here 1. Don't show the menu at all 2. Update the regex to handle this text and show "No results"

Ideally close the menu I think

> Suggestion - When the BlockMenu is active, we should handle Shit-Tab to move select upwards.

Good idea

> Inside the table, The highlight menu has two seperators.

Buggy bug
